README.md

```diff
@@ -165,6 +165,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     The Women in Tech Summit inspires, educates and connects women in the technology industry.
 
+  - [Wonder Women Tech Summit](https://wonderwomentech.com/)
+
+    Wonder Women Tech’s mission is to highlight, celebrate and educate women and the underrepresented in STEAM industries. We create conferences, events and workshops where we bring together changemakers, pioneers, innovators, and high-level speakers and attendees to explore ideas, discussions, and solutions for women and the underrepresented to succeed and thrive
```

Comment:
Remove the section, 'we create'. The list is not representing them nor marketing them. This has to be more information based and basically just creates awareness. Slightly rework it and it's good to go.

Thanks for the PR 👍